ui-modules/catalog/app/views/bundle/type/type.template.html

```diff
@@ -20,8 +20,8 @@
     <header class="row">
         <div class="col-md-12">
             <div class="media">
-                <div class="media-left">
-                    <img ng-src="{{type | iconGeneratorPipe:'symbolicName'}}" class="media-object" alt="{{::type.symbolicName}}'s logo" />
+                <div class="media-left" ng-if="type.symbolicName">
```

Comment:
`ng-if` is needed to prevent `::type | ...` being evaluated when `type` is set but not populated, also elsewhere